api/src/test/java/org/openmrs/api/impl/ProgramWorkflowServiceImplTest.java

```diff
@@ -0,0 +1,76 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.api.impl;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import static org.mockito.Mockito.verify;
+import org.openmrs.ProgramAttributeType;
+import org.openmrs.api.ProgramWorkflowService;
+import org.openmrs.api.db.ProgramWorkflowDAO;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+@RunWith(PowerMockRunner.class)
+public class ProgramWorkflowServiceImplTest {
```

Comment:
These tests need to be in ProgramWorkflowServiceTest, you don't need this new class